README.md

```diff
@@ -60,48 +60,23 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [HELLO-NEW] <a name="about-project">Html/css</a>
 
-> Describe your project in 1 or 2 sentences.
+>  this is the first phase of my first portfolio project
```

Comment:
- [ ] As suggested [here](https://github.com/stino-x/HELLO-NEW/pull/2#pullrequestreview-1247915815), please consider removing this comment. It is a mere template as guide on how to fill your information in.